src/client/lib/hooks/useRemoveEncryptedEmailParam.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { removeEcryptedParam } from './useRemoveEncryptedEmailParam';
+
+describe('useRemoveEncryptedEmailParam', () => {
+  it('deleted EcryptedEmail param from query string', () => {
+    expect(removeEcryptedParam('abc=def&encryptedEmail=abc&test=def')).toBe(
```

Comment:
Small typo on this `removeEcryptedParam`

Also, this isn't rendering the hook itself just testing that method, so probably doesn't have to be `.tsx` just `.ts`